.github/workflows/presubmit-compositions.yaml

```diff
@@ -23,7 +23,48 @@ on:
     branches: ["master"]
     paths:
       - "experiments/compositions/**"
+
+env:
+  GOWORK: off
 jobs:
+  verify-goimports:
```

Comment:
could we rely on the golanci-lint uberlinter to run `goimports`? Can be done as a follow on

Reply:
Will do. We have one more CL in progress: 
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2493

The top level linter is not catching anything in compositions.